provider/common/bootstrap_test.go

```diff
@@ -135,6 +135,55 @@ func (s *BootstrapSuite) TestCannotStartInstance(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, "cannot start bootstrap instance: meh, not started")
 }
 
+func (s *BootstrapSuite) TestBootstrapSeries(c *gc.C) {
+	s.PatchValue(&jujuversion.Current, coretesting.FakeVersionNumber)
+	s.PatchValue(&series.HostSeries, func() string { return "precise" })
+	stor := newStorage(s, c)
+	checkInstanceId := "i-success"
+	checkHardware := instance.MustParseHardware("arch=ppc64el mem=2T")
+
+	startInstance := func(
+		_ string, _ constraints.Value, _ []string, _ tools.List, icfg *instancecfg.InstanceConfig,
+	) (
+		instance.Instance, *instance.HardwareCharacteristics, []network.InterfaceInfo, error,
+	) {
+		return &mockInstance{id: checkInstanceId}, &checkHardware, nil, nil
+	}
```

Comment:
Can you please reformat this? Typically if you want a multi-line function, you align it as you would a struct.